Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM iron/base
+FROM alpine:3.4
 
 COPY ./deployments /usr/bin/
 
```

Comment:
We'll need to install ca-certificates package, otherwise using S3 (the real one) will not work.

```
RUN apk update && apk upgrade \
     apk add ca-certificates && \
     rm -rf /var/cache/apk/*
```